support/cas-server-support-aws-cognito-authentication/src/test/java/org/apereo/cas/config/AmazonCognitoAuthenticationConfigurationTests.java

```diff
@@ -0,0 +1,58 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.authentication.AuthenticationHandler;
+import org.apereo.cas.config.support.CasWebApplicationServiceFactoryConfiguration;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.logout.config.CasCoreLogoutConfiguration;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.autoconfigure.RefreshAutoConfiguration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * This is {@link AmazonCognitoAuthenticationConfigurationTests}.
+ *
+ * @since 6.3.3
+ */
+@SpringBootTest(classes = {
+    RefreshAutoConfiguration.class,
+    CasCoreAuthenticationPrincipalConfiguration.class,
+    CasCoreAuthenticationConfiguration.class,
+    CasCoreAuthenticationHandlersConfiguration.class,
+    CasCoreAuthenticationSupportConfiguration.class,
+    CasCoreAuthenticationServiceSelectionStrategyConfiguration.class,
+    CasWebApplicationServiceFactoryConfiguration.class,
+    CasCoreServicesConfiguration.class,
+    CasCoreTicketsConfiguration.class,
+    CasCoreTicketIdGeneratorsConfiguration.class,
+    CasCoreTicketCatalogConfiguration.class,
+    CasCoreUtilConfiguration.class,
+    CasCoreWebConfiguration.class,
+    CasCoreHttpConfiguration.class,
+    CasPersonDirectoryConfiguration.class,
+    CasCoreLogoutConfiguration.class,
+    CasCoreConfiguration.class,
+    CasCoreNotificationsConfiguration.class,
+    AmazonCognitoAuthenticationConfiguration.class
+}, properties = {
+    "cas.authn.cognito.mapped-attributes[0].name=custom:eduPersonUniqueId",
```

Comment:
Remove the class, move settings over to Handler class

Reply:
I will do that.